gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -198,9 +198,12 @@ func loadStdlibPackage(pkgPath, stdlibDir string, store gno.Store) {
 	m.RunMemPackage(memPkg, true)
 }
 
-type gnoStoreContextKeyType struct{}
+type keeperContextKey string
```

Comment:
Would it be simpler to define a separate type for each? Like
```go
type gnoStoreContextKey struct{}
type customContextKey struct{}
```
And then use them like `customContextKey{}` in the code and omit any extra variable and constant declarations altogether?

Reply:
I think constant singletons make sense here IMO; the same is suggested for instance in the [go blog `context` article](https://go.dev/blog/context). Maybe I can change them to integer constants/